mne/viz/backends/_pyvista.py

```diff
@@ -320,3 +325,19 @@ def _get_view_to_display_matrix(size):
 
 def _close_all():
     pass
+
+
+def _check_notebook():
+    if _run_from_ipython():
+        try:
+            return type(get_ipython()).__module__.startswith('ipykernel.')  # noqa
+        except NameError:
+            return False
+
+
+def _run_from_ipython():
+    try:
+        py = __IPYTHON__  # noqa
+        return True
+    except NameError:
+        return False
```

Comment:
Maybe this can be done more simply as `return '__IPYTHON__' in globals()`? Not sure if it's better or worse. But at the very least, I don't see why you would assign to `py` rather than just doing `__IPYTHON__` directly (it seems to be what the ipython docs do in my minimal searches)

Reply:
Surprisingly, both `__IPYTHON__ in globals()` and `globals().get('__IPYTHON__', False)` returns `False` inside a jupyter notebook in my testing.
I'll modify to use `__IPYTHON__` directly, thank you.